src/aggregate/aggregate_request.c

```diff
@@ -568,6 +569,12 @@ static int parseQueryArgs(ArgsCursor *ac, AREQ *req, RSSearchOptions *searchOpts
     }
   }
 
+  // In dialect 2, we require a non empty numeric filter
+  if (req->reqConfig.dialectVersion >= 2 && isEmptyFilterValue){
+      QERR_MKBADARGS_FMT(status, "Invalid numeric filter");
```

Comment:
might also be a `GEOFILTER`
```suggestion
      QERR_MKBADARGS_FMT(status, "Invalid numeric/geo filter value/s");
```